src/view/constants.ts

```diff
@@ -6,6 +6,10 @@ export const CONSTANTS = {
     CURRENTLY_RUNNING: (file: string) => {
         return `Currently running: ${file}`;
     },
+    DEVICE_NAME: {
+        CPX: "CPX",
+        MICROBIT: "Micro:bit",
```

Comment:
Case, all trademark and marketing info use "micro:bit" no cap...  Do we need to conform?

Reply:
Good one, we should definitely conform to their branding